### PR TITLE
Working thru a small list of cosmetic changes...

### PR DESCRIPTION
**Pull request by sawtoothgeek**
First of a set of cosmetic changes.

### CODE REVIEW

**Comment by sawtoothgeek** [comment]
Reopened after I pushed the latest set of cosmetic changes...and fixes. Obfuscation checkbox on the send tab/page has been hidden too until we enable obfuscation application wide.

**Comment by sawtoothgeek** [comment]
Hey there Dan...

I had a system meltdown on my openSuse box :( have been immersed in 
rebuilding things since. Going to have the latest openSuse or Ubuntu 
reinstalled so now I will have the latest linux distro.

That must be in main.cpp, you did not mention the module. I had a 
problem with those asserts triggering one day and had to disable them. I 
was going to re-enable those and wanted to ask you if you guys were 
seeing those asserts triggering too.

That's really about it on that.

*I would say hold off on merging my recent commits all together into 
master until I am back up and running.

*I had to commit and push when I was only half done, I did that just 
before my system melted down...the system disks partition table went 
into lulu land.

So bottom line the color theme and scaling fixes are not all in yet and 
not fully tested too, so do not merge those commits in yet I will update 
them tonite if all goes well.

Thanks, let me know if you need anything...I will check email here a lil 
later on.
Steve







ATC wrote:

**Comment by atcsecure** [comment]
Yeah Don't disable those as it checks block chain on disk, if your copy is
corrupting you need to resync

I am afk now but can explain more later

Thanks

Dan

**Comment by sawtoothgeek** [comment]
A roger on that Dan, and thanks for the quick info on that. I had a 
feeling it was something along those lines going on. I will re-enable 
those checks/asserts ASAP with my very next commit.

Thanks again!
Steve




ATC wrote:
> Yeah Don't disable those as it checks block chain on disk, if your copy is
> corrupting you need to resync
>
> I am afk now but can explain more later
>
> Thanks
>
> Dan
>

**Comment by sawtoothgeek** [comment]
Hi Dan...still messing around with setting up my dev machine...

I found out my motherboard had some issues and wrong bios settings. 
Somehow they changed, this box "megapower" overall has been solid as a 
rock with Suse 13.1 and Windows 7 Pro 64bit for over 3 years. The first 
hiccup happened three months ago tho when I was testing out my latest 
version of an AI system called HAL9000 that I am developing in my spare 
time written C++. It was being developed in windows and I just added a 
new thread to monitor just the sound sensor (mic) for the brain. When I 
ran it the system worked well and the sound was coming in realtime and 
being plotted to an o-scope like lil graphing widget I wrote a ways 
back. SO I was amazed how the extra threads made the system run more 
realtime and full duplex. THEN Windows locked up all of sudden while it 
was running. Seconds later the blue screen of death appeared. Upon 
rebooting I had found out that windows had proceeded to destroy itself 
and would not boot in all the way anymore. So I re-installed windows in 
that partition which retained it's data. That was three months ago.

Ever since then the dev box "megapower" seemed a tad a unstable have had 
reboots and lock ups now and then when there were none before. I am a 
bit pissed cause this system has 32GB of ram and a powerful AMD FX4100 
Quad core etc. Radeon 7870 2GB video. I know all this is old news now as 
I custom built this system up three years ago but it is still not that 
old. So now I am having issues with the MB itself and getting a bit 
bogged down. I may setup my other dev box as a dev machine for 
BlocknetDX dev if "megapower" is not ready by today.

Anyway that is why I am AWOL for now, lol. Wish me luck...I will be back 
up and working on BlocknetDX later on today or early morning tomorrow at 
the latest.

Oh by the way I tried the latest openSuse 42.3 and was more impressed by 
my latest look at Ubuntu 17 sooooooo, I am going to go with ubuntu and 
that way I will be more compat with the team too. In my opinion ubuntu 
is a bit lighter weight and quicker to install and nicer to look at and 
use too, right out of the box.

Cheers,
Steve





ATC wrote:
> Yeah Don't disable those as it checks block chain on disk, if your copy is
> corrupting you need to resync
>
> I am afk now but can explain more later
>
> Thanks
>
> Dan
>
